config/initializers/graphql_cache.rb

```diff
@@ -0,0 +1,36 @@
+GraphQL::Cache.configure do |config|
+  config.namespace = 'compliance' # Cache key prefix for keys generated by graphql-cache
+  config.cache     = Rails.cache      # The cache object to use for caching
+  config.logger    = Rails.logger     # Logger to receive cache-related log messages
+  config.expiry    = 15552000         # 6 months (in seconds)
```

Comment:
We should probably discuss this and even think more seriously about introducing an LRU cache redis instance directly. Fields are cached alongside its updated_at, e.g: `compliance:Rule:rules/30b9d4c7-4924-4996-bd89-7e29117fccf7-20200227230834209106:title` to force us to realize whether the key can be used or not. It's a sort of 'automatic invalidation' as any time a rule is updated, the old entries will be useless.

Reply:
So the old entries with older `updated_at` in the key are kept in redis?

Reply:
Correct, that's why I'd want to go with an instance like https://redis.io/topics/lru-cache

Reply:
Correct, that's why I'd want to go with an instance like https://redis.io/topics/lru-cache